### DISCUSSION TITLE
problem in isce proccesing  ; step7 pairs_misreg

### DISCUSSION THREAD

**Discussion by Matin-GorganSaraj** [alexisInSAR/EZ-InSAR#50] [Help]
hi alexis 
i have a issue with isec step7 for slc stack ; in this step i have a warning that isce gives me , bad match at level 1 
i dont know how this warning effects me and i need you help about it 
i look after this problem in isce2 issue but i don get any about except ;this https://github.com/isce-framework/isce2/issues/259
in this page they talking about this exact issue but they haven't the solotion.
another problem is in this step i locked in middle of it and i dont know how i fix this issue.
this is terminal in this step
![Screenshot (204)](https://github.com/alexisInSAR/EZ-InSAR/assets/130948662/556704e5-7311-499a-b10f-7f3054dea6b6)
and suddenly the warning pop ups!
![Screenshot (206)](https://github.com/alexisInSAR/EZ-InSAR/assets/130948662/686509f5-bee6-4f40-a4ca-57d73dafc61c) 
With all regards
Matin

**Comment by alexisInSAR** [comment]
Hi, 

I think you can try to continue the processing. 

Alexis
